System/RTETools.h

```diff
@@ -216,6 +216,17 @@ namespace RTE {
 	/// <param name="angleRadians">The angle in radians to be converted.</param>
 	/// <returns>The converted angle in degrees.</returns>
 	inline float RadiansToDegrees(float angleRadians) { return angleRadians / c_PI * 180.0F; }
+
+	/// <summary>
+	/// Rounds a float to a set fixed point precision (digits after decimal point) with option to always ceil or always floor the remainder.
+	/// </summary>
+	/// <param name="inputFloat">- The input float to round.</param>
+	/// <param name="roundingMode">- (0, 1, 2) What method of rounding to use for the result. 
+	/// <para>0 (default option) - System default.</para>
```

Comment:
Man this looks horrible.
Just do 
`/// <param name="roundingMode">Method of rounding to use. 0 for system default, 1 for floored remainder, 2 for ceiled remainder.</param>`.